core/src/main/scala/trees/Trees.scala

```diff
@@ -230,6 +230,22 @@ object Tree {
       case TopType => TopType
     }
   }
+  
+  def replaceAndCount(id: Identifier, v: Tree, body: Tree)(implicit rc: RunContext): (Tree, Int) = {
```

Comment:
`replace`-like methods could be rewritten to be more like this

Reply:
Yes thanks.